### ISSUE TITLE
Will no longer work with latest cake.

### ISSUE THREAD

**Issue by waynebrantley** [cake-contrib/Cake_Git#58]
```
Error: The assembly 'Cake.Git, Version=0.16.1.0, Culture=neutral, PublicKeyToken=null' 
is referencing an older version of Cake.Core (0.22.0). 
This assembly need to reference at least Cake.Core version 0.26.0. 
Another option is to downgrade Cake to an earlier version. 
It's not recommended, but you can explicitly opt-out of assembly verification 
by configuring the Skip Verification setting to true
(i.e. command line parameter "--settings_skipverification=true", 
environment variable "CAKE_SETTINGS_SKIPVERIFICATION=true", 
read more about configuration at https://cakebuild.net/docs/fundamentals/configuration)
Command exited with code 1
```

**Comment by waynebrantley** [comment]
version 0.26.0 released on 2/26/18 is the one it does not work with.
Of course It still works with 0.25.0

**Comment by devlead** [comment]
This is fixed by 0.17.0
